content/organizations/managing-saml-single-sign-on-for-your-organization/troubleshooting-identity-and-access-management-for-your-organization.md

```diff
@@ -27,7 +27,7 @@ If an organization member has missing SCIM metadata, then you can re-provision S
 
 If you suspect or notice that any users are not provisioned or deprovisioned as expected, we recommend that you audit all users in your organization.
 
-To check whether users have a SCIM identity (SCIM metadata) in their external identity, you can review SCIM metadata for one organization member at a time on {% data variables.product.prodname_dotcom %} or you can programatically check all organization members using the {% data variables.product.prodname_dotcom %} API.
+To check whether users have a SCIM identity (SCIM metadata) in their external identity, you can review SCIM metadata for one organization member at a time on {% data variables.product.prodname_dotcom %} or you can programmatically check all organization members using the {% data variables.product.prodname_dotcom %} API.
```

Comment:
#### Before
https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/troublesho
<img width="735" alt="image" src="https://github.com/github/docs/assets/2119212/e02af4e7-c65c-4389-ae10-b783e57b5358">
oting-identity-and-access-management-for-your-organization#auditing-users-for-missing-scim-metadata
#### After
https://docs-30890-94eb78.preview.ghdocs.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/troubleshooting-identity-and-access-management-for-your-organization#auditing-users-for-missing-scim-metadata
<img width="706" alt="image" src="https://github.com/github/docs/assets/2119212/9187bd78-bc20-4b4f-8d0a-4bd84290fd3c">